fastai/vision/augment.py

```diff
@@ -191,12 +195,19 @@ def encodes(self, x:(Image.Image,TensorBBox,TensorPoint)):
 class RandomCrop(RandTransform):
     "Randomly crop an image to `size`"
     split_idx,order = None,1
-    def __init__(self, size, **kwargs):
+    def __init__(self,
+        size:(int, tuple), # Size to crop to, duplicated if one value is specified
+        **kwargs
+    ):
         size = _process_sz(size)
         store_attr()
         super().__init__(**kwargs)
 
-    def before_call(self, b, split_idx):
+    def before_call(self,
+        b,
+        split_idx:int # Index of the train/valid dataset
+    ):
+        "Randomly positioning crop if train dataset (split_idx=0) else center crop"
```

Comment:
```suggestion
        "Randomly positioning crop if `split_idx==0` else center crop"
```

Reply:
@muellerzr why not just "Randomly positioning crop if training, otherwise center crop "

Then we leave the `split_idx` information in the document for the param, as `split_idx` is more of an implementation detail of how Dataloaders work with transforms. I wouldn't expect most users to know that `split_idx==0` means we are working with the training set.

Reply:
@muellerzr why not keep train dataset instead? I think that would be more familiar to the majority of users, making the documentation easier to digest. Knowing what split_idx is can be left to the documentation for that specific param.

Reply:
@dienhoa had a conversation with zach, go with my recommendation and we can get this merged. So use: "Randomly positioning crop if train dataset else center crop"

Reply:
Done. Thanks @marii-moe !